### ISSUE TITLE
T-10 reqirement

### ISSUE THREAD

**Issue by B-30F9** [PygmalionOfCyprus/cmo-db-requests#1494]
### DB Selector

Both

### Hypothetical

Yes

### Name

T10 Flanker-A

### Country

USSR

### Service

Air Force

### Number in Service

9

### Length (m)

19.5 (64 feet)

### Wingspan (m)

14.6 (48 feet)

### Height (m)

5.7 (19 feet)

### Empty Weight (kg)

12100 (26676 lb)

### Maximum Takeoff Weight (MTOW) (kg)

25740 (66139 lb)

### Crew

1

### Propulsion

2x AL-21F3(izdyelie 89)(T10-1/T10-2)/AL-31F(izdyeliye 99)(T10-3/T10-4)/AL-21F3(izdyelie 89)(T10-5/T10-6/T10-9/T10-10/T10-11)

### Service Ceiling (ft)

57415 ft

### Military/Afterburner Speed (kt)

1385.7 mph

### Cruise Speed (kt)

？

### Supercruise

Yes

### Ferry Range (nm)

？

### Sensors

![image](https://user-images.githubusercontent.com/107861955/175896272-0b5a3428-bfaa-4c1f-9a51-e522410f1427.png)

### Mounts

![image](https://user-images.githubusercontent.com/107861955/175896395-0c444d70-45d8-4a34-8e1e-70286aeff463.png)

### Loadout Options

?

### Sources

https://www.armedconflicts.com/Sukhoi-T-10-Suhoy-T-10-t9002

**Comment by KLABDB3K** [comment]
Pretty sure T10 never had sensors bar the experimental radar and weapon trials were carried out with the late T10 peototpye which was virtually a vanilla SU-27 so hardly worth a DB slot?

**Comment by PygmalionOfCyprus** [comment]
Yeah, prototype aircraft really aren't worth adding if they just became a real aircraft (already in the DB) later. If it was a prototype for some zany platform that didn't make it to service that'd be something else, but...
